lascli/parser/predictions.py

```diff
@@ -58,6 +58,7 @@ def create_predictions_parser(subparsers):
     list_predictions_parser.add_argument('--next-token', '-n', default=None)
     list_predictions_parser.add_argument('--sort-by', choices={'createdTime'})
     list_predictions_parser.add_argument('--order', choices={'ascending', 'descending'})
+    list_predictions_parser.add_argument('--model-id')
```

Comment:
We do not need default to be `None` as with the other parameters?

Reply:
`default` defaults to `None`. It's rather the other arguments that do not need to specify `default=None` imo. But I don't think refactoring that is necessary for this PR. What do you think?

Reply:
I agree, refactoring should be a separate PR. Not sure it is needed at all though.